packages/flutter_tools/lib/src/ios/device_ios.dart

```diff
@@ -292,6 +295,61 @@ class IOSSimulator extends Device {
   }
 
   @override
+  bool isSupported() {
+    if (!Platform.isMacOS) {
+      _supportMessage = "Not supported on a non Mac host";
+      return false;
+    }
+
+    // Step 1: Check if the device is part of a blacklisted category.
+    //         We do not support WatchOS or tvOS devices.
+
+    RegExp blacklist = new RegExp(r'Apple (TV|Watch)', caseSensitive: false);
+
+    if (blacklist.hasMatch(name)) {
+      _supportMessage = "Flutter does not support either the Apple TV or Watch. Choose an iPhone 5s or above.";
```

Comment:
You also have the name right here, you could use the name instead of hardcoding.

"Flutter does not yet support $name, sorry."

You could even link to the FAQ if you like.  http://flutter.io/faq/#what-devices-and-os-versions-does-flutter-run-on